blue-note/theme.json

```diff
@@ -0,0 +1,224 @@
+{
+	"$schema": "https://schemas.wp.org/wp/6.2/theme.json",
+	"customTemplates": [
+		{
+			"name": "single-no-sidebar",
+			"postTypes": [
+				"page",
+				"post"
+			],
+			"title": "Single, no sidebar"
+		}
+	],
+	"settings": {
+		"appearanceTools": true,
+		"color":{
+			"palette": [
+				{
+					"slug": "base",
+					"name": "Base",
+					"color": "#F5F3EA"
+				},
+				{
+					"slug": "contrast",
+					"name": "Contrast",
+					"color": "#000000"
+				},
+				{
+					"slug": "primary",
+					"name": "Primary",
+					"color": "#000000"
+				},
+				{
+					"slug": "secondary",
+					"name": "Secondary",
+					"color": "#6E6E6E"
+				},
+				{
+					"slug": "tertiary",
+					"name": "Tertiary",
+					"color": "#ADADAD"
+				}
+			],
+			"duotone": [
+				{
+					"slug": "duotone-1",
+					"name": "Duotone 1",
```

Comment:
I think it's more helpful for the user if these are a bit more descriptive like "black and red" for example

Reply:
yes, ggreed, but I did not want to make the decision alone =D

Reply:
The design only had one color per duotone, so I kept the dark color black.